pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -56,15 +61,18 @@ func LowNodeUtilization(ctx context.Context, client clientset.Interface, strateg
 		return
 	}
 	targetThresholds := strategy.Params.NodeResourceUtilizationThresholds.TargetThresholds
-	if !validateTargetThresholds(targetThresholds) {
+	if !validateThresholds(targetThresholds) {
```

Comment:
Better but not sufficient. `thresholds` has to be less than `targetThresholds` per a resource type. @lixiang233  Can you add check for that as well?

Also, `thresholds` and `targetThresholds` have gave me a lot of headache before learning what they actually mean. I would prefer to rename both of them into `lowThreshold` and `highThreshold` at least on the code level. Also given the strategy is configured as:
```
  "LowNodeUtilization":
     enabled: true
     params:
       nodeResourceUtilizationThresholds:
         thresholds:
           "cpu" : 20
           "memory": 20
           "pods": 20
         targetThresholds:
           "cpu" : 50
           "memory": 50
           "pods": 50
```
we can drop `thresholds` and `targetThresholds` and have:
```
  "LowNodeUtilization":
     enabled: true
     params:
       nodeResourceUtilizationThresholds:
         low:
           "cpu" : 20
           "memory": 20
           "pods": 20
         high:
           "cpu" : 50
           "memory": 50
           "pods": 50
```

@damemi @seanmalloy anything against?

Reply:
Should we allow `thresholds` and `targetThresholds` configuring different resources? For instance, `thresholds` configured `cpu` and `mem` while `targetThresholds` only configured `pods`.

Reply:
Good point. Assuming low threshold has a cpu set but high does not, you mark a node as underutilized wrt. cpu. You then find over-utilized node(s) wrt. memory. In the process of moving workload from over-utilized (since you want to decrease memory consumption), you move some pods to the underutilized nodes with a hope there's at least some memory left on those nodes. So, in practice this scenario is possible though you are removing any guarantee the list of underutilized nodes is selected effectively. I.e. the nodes have sufficient memory left. Since there's no low threshold defaulting for memory, you might consume all the memory left and end up with nodes which are memory over utilized. Also, selecting under-utilized nodes based on cpu becomes irrelevant since you then may pick any subset of nodes since you still have no guarantee there's sufficient memory left.

So, the answer is no.